### PR TITLE
IMU orientation changes in importIitYarp.

### PR DESCRIPTION
**Pull request by simbamford**

### CODE REVIEW

File: python/libraries/bimvee/importIitYarp.py [hunk 1]
```diff
@@ -240,23 +240,23 @@ def samplesToImu(inDict, **kwargs):
             tsOut[imuPtr] = ts # Just take the first ts for a group of samples
         sensorPrev = sensor
         if sensor == 0:
-            acc[imuPtr, 0] = value
+            acc[imuPtr, 1] = -value
```
Comment (arrenglover) [inline python/libraries/bimvee/importIitYarp.py:243]: this should be + (x axis is index 1 but positive)

File: python/libraries/bimvee/importIitYarp.py [hunk 2]
```diff
@@ -240,23 +240,23 @@ def samplesToImu(inDict, **kwargs):
...
         elif sensor == 1:
-            acc[imuPtr, 1] = value
+            acc[imuPtr, 0] = value
```
Comment (arrenglover) [inline python/libraries/bimvee/importIitYarp.py:245]: this should be negative (y axis is sensor 0 and -ive

File: python/libraries/bimvee/importIitYarp.py (hunk 1 above)
Comment (arrenglover) [reply python/libraries/bimvee/importIitYarp.py:243]: sorry

Comment (simbamford) [reply python/libraries/bimvee/importIitYarp.py:243]: No - the sensor value from the raw data comes from line 242 (=0). the index in line 243 is the array that the value is being put into, where 0=x, 1=y, 2=z.

**Pull request merged**